Documentation/DocuBlocks/Rest/Administration/post_admin_cluster_rebalance.md

```diff
@@ -4,25 +4,29 @@
 
 @RESTHEADER{POST /_admin/cluster/rebalance, Compute a set of move shard operations to improve balance.}
```

Comment:
The summary seems too long and not end-user focused. A more detailed explanation should go into the `@RESTDESCRIPTION`, which every DocuBlock should have.

The wording leaves it open whether the endpoint actually executes a rebalancing or if it only analysis the situation and proposes how to move shards.